homeassistant/components/transmission/strings.json

```diff
@@ -4,10 +4,12 @@
       "user": {
         "title": "Set up Transmission Client",
         "data": {
+          "ssl": "[%key:common::config_flow::data::ssl%]",
           "host": "[%key:common::config_flow::data::host%]",
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]",
-          "port": "[%key:common::config_flow::data::port%]"
+          "port": "[%key:common::config_flow::data::port%]",
```

Comment:
Please add the attribute sorted correctly

Reply:
Hey @edenhaus do you mean alpabatically sorted? Because if that's the case, I need to sort also entries that I did not add. 

If there is another sorting method being used, can you share me where to find this? Because the docs don't seem to mention one

Reply:
Yes, please sort existing items, existing and the ones you add, alphabetically 👍

Reply:
Did this!